canonical-websites/mir-server.io
# Build redesigned homepage with latest Vanilla

## Done

- Set up project with node-sass and Vanilla 1.8.0
- Rewrote index.html with latest Vanilla styles

## QA

- `./run`
- Navigate to http://0.0.0.0:8028/
- Check that the homepage looks good
- Check that it matches the [copydoc](https://docs.google.com/document/d/1wqmWPx4TJ_oJoFt4gBOg4trYMIAsugMYtHGXzYJx8kc/edit#)
- Run through the [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)

## Issues

Fixes ubuntudesign/web-squad#746
